api-server/src/roster/urls.py

```diff
@@ -25,6 +25,13 @@
     path('aviators/fromuser/<int:user_id>/', AviatorFromUserListView.as_view()),
     path('prospective_aviators/detail/', ProspectiveAviatorDetailView.as_view()),
     path('stats/', StatsView.as_view()),
+    path('stats/flightlog/list/<aviator_pk>/', FlightLogListView.as_view()),
+    path('stats/flightlog/aggregate/<aviator_pk>/', FlightLogAggregateView.as_view()),
+    path('stats/flightlog/timeseries/<aviator_pk>/', FlightLogTimeSeriesView.as_view()),
+    path('stats/flightlog/timeseries/90days/<aviator_pk>/', FlightLogTimeSeries90DaysView.as_view()),
+    path('stats/combatlog/list/<aviator_pk>/', CombatLogListView.as_view()),
+    path('stats/combatlog/aggregate/<aviator_pk>/', CombatLogAggregateView.as_view()),
+    path('stats/combatlog/timeseries/<aviator_pk>/', CombatLogTimeSeriesView.as_view()),
```

Comment:
@afuggetta do we also want to do the time series on kills for the front page? I can be convinced either way.

Reply:
yeah we can do that, we can fill that space up. I'll work on it

Reply:
unless you want to take the backend and I'll take the front-end

Reply:
ill do the backend when i generalize the other time series

Reply:
sounds good